collect_app/src/androidTest/java/org/odk/collect/android/tasks/FormLoaderTaskTest.java

```diff
@@ -14,13 +15,19 @@
 
 import java.io.File;
 import java.util.Arrays;
+import java.util.Collections;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.notNullValue;
 
 public class FormLoaderTaskTest {
```

Comment:
I'd love to say "this should be Robolectric" but I think that would involve picking the task apart. The `XTaskTest` classes are pretty much the prime candidates for moving out of Espresso so I'm confident that will happen later.

Reply:
Yeah, agreed that coming back to this sounds good. 

I think the new Robolectric tests may actually offer sufficient coverage. There is something comforting about knowing that these are even more end-to-end, though.